internal/main.go

```diff
@@ -148,8 +148,8 @@ func getMetadataProvider(providerName string) (func() (providers.Metadata, error
 		return packet.FetchMetadata, nil
 	case "openstack-metadata":
 		return openstackMetadata.FetchMetadata, nil
-	case "vagrant-virtualbox":
-		return vagrant_virtualbox.FetchMetadata, nil
+	case "virtualbox":
```

Comment:
Can you also add `"vagrant-virtualbox"` to this line? Otherwise won't coreos-metadata will stop working for users when they upgrade to this version?